src/main/java/org/motechproject/newebodac/mapper/VaccineeMapper.java

```diff
@@ -14,6 +15,7 @@
 import org.motechproject.newebodac.domain.Vaccinee;
 import org.motechproject.newebodac.domain.enums.EntityType;
 import org.motechproject.newebodac.dto.VaccineeDto;
+import org.motechproject.newebodac.helper.EncryptionHelper;
 
 @Mapper(uses = { ExtraFieldMapper.class }, unmappedTargetPolicy = ReportingPolicy.IGNORE)
 public interface VaccineeMapper extends EntityMapper<VaccineeDto, Vaccinee> {
```

Comment:
change this interface to an abstract class and add autowired EncryptionHelper here